fms/models/roberta.py

```diff
@@ -331,10 +331,10 @@ def _hf_sd_to_fms_sd(hf_sd: Mapping[Any, Any]) -> Mapping[Any, Any]:
         if name == "roberta.embeddings.position_embeddings.weight":
             new_sd[new_name] = new_sd[new_name][2:]
 
-    fused_sd = _convert_fused_qkv_0_0_4(new_sd)
+    fused_sd = __v0_0_4_adapter(new_sd)
 
     return fused_sd
 
 
 serialization.register_adapter("roberta", "hf", _hf_sd_to_fms_sd)
-serialization.register_adapter("roberta", "fms.v0.0.4", _convert_fused_qkv_0_0_4)
+serialization.register_adapter("roberta", "fms.v0.0.4", __v0_0_4_adapter)
```

Comment:
this naming might be confusing. if I have weights saved from a version of fms _older_ than 0.0.4 that need to be adapted, I wouldn't expect that the model "source" I request will be called fms.v0.0.4.